typings/device-overall-status.d.ts

```diff
@@ -0,0 +1,19 @@
+/* tslint:disable:no-namespace */
+declare namespace DeviceOverallStatus {
```

Comment:
Do we really need to use a namespace? We also do use plain type exports in other typings that get re-exported by the main balena-sdk typings.